packages/venia-concept/src/components/Main/main.css

```diff
@@ -2,7 +2,6 @@
     background-color: white;
     color: rgb(var(--venia-text));
     position: relative;
-    z-index: 1;
```

Comment:
This causes the main menu to appear incorrectly (without back button). Can we add this back? Was it removed intentionally?

Reply:
This was intentional. Up until now, all modals were a child of the main container and the z-index worked as intended. 

But now the modal is part of the content which is nested deep into main container, so we had to remove the z-index on the main wrapper and add them to individual drawers. I have fixed the CSS issues and now the drawers appear correctly.

Reply:
Thanks for the explanation! I did not know the history of the modals :)